notebooks/signac_105_Command_Line_Interface.ipynb

```diff
@@ -6,6 +6,13 @@
    "source": [
```

Comment:
<p>Make the note </p><p> </p><h2>just a bit bolder</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
This is how it renders in Jupyter, just not on ReviewNB.

![image](https://user-images.githubusercontent.com/3943761/107470652-5a733000-6b31-11eb-9012-3dcaf54ffc1c.png)